flutter/lib/src/uvccamera_controller.dart

```diff
@@ -139,7 +139,7 @@ class UvcCameraController extends ValueNotifier<UvcCameraControllerState> {
     _isInitializedOrDisposed();
 
     if (value.isRecordingVideo) {
-      throw Exception('UvcCameraController is already recording video');
```

Comment:
It's incorrect to ask for recording to start and do nothing when it's already started. The integrating app should ensure that state is correct and not to call that a second time. On top of that, silently ignoring the request is error prone - what if the requested recording mode second time is different. If illegal state exception causes issues for the integrating app - the app developer should take care of the faulty app logic.

Reply:
Agree